### ISSUE TITLE
[TextInput] Android/iOS Keyboard next/prev Buttons & tab Support

### ISSUE THREAD

**Issue by kietnguyen0814** [facebook/react-native#35077]
### Description

I have been unable to find throughout the documentation, or my own experiments a way to enable Android/iOS keyboard navigation. I feel like this is important and expected from users to be able to navigate between input elements and submit from the Android/iOS keyboard.

If somebody already has a solution for this, please let me know 😄

Request

Activate Android next/prev on the Button and other user input elements
Enable grouping of input elements together as a "form"

### Version

0.67.3

### Output of `npx react-native info`

System:
OS: macOS 12.6
CPU: (12) ×64 Intel(R) Core (TM) 17-9750H CPU @ 2.60GHz
Memory: 595.79 MB / 16.00 GB
Shell: 5.8.1 - /bin/zsh
Binaries:
Node: 14.18.1 - /usr/local/bin/node
Yarn: 1.22.18-~/.node _modules/bin/yarn
Watchman: Not Found
Managers:
CocoaPods: Not Found
SDKs:
iOS SDK:
Platforms: DriverKit 21.4, 10S 15.5, macOS 12.3, tvOS 15.4, watchos 8.5
Android SDK:
API Levels: 29, 33
Build Tools: 29.0.2, 30.0.1, 30.0.2, 30.0.3, 31.0.0
System Images: android-28 | Google APIs Intel x86 Atom, android-30 | Google APIs Intel x86 Atom, android-30 | Goo
gle Play Intel x86 Atom
Android NDK: 22.1.7171670
IDES:
Android Studio: 2020.3 AI-203.7717.56.2031.7678000
Xcode: 13.4.1/13F100 - /usr/bin/xcodebuild
Languages:
Java: 1.8.0_181 - /usr/bin/javac
npmPackages:
@react-native-community/cli: Not Found
react: Not Found
react-native: Not Found
react-native-macos: Not Found
npmGlobalPackages:
*react-native*: Not Found

### Steps to reproduce

Create a form with button
Run on device
Connect to physical keyboard
Try to press tab/shift+tab

Expect result:
Can focus to next component(TextField, Button, ...) when user press tab from physical keyboard

### Snack, code example, screenshot, or link to a repository

https://github.com/facebook/react-native/issues/641

**Comment by Polad10** [comment]
+1
